googlecloud-plugins/widgets/BigQuery-batchsource.json

```diff
@@ -0,0 +1,48 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "BigQuery",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
+          "name": "projectId"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "JSON Key File Path",
+          "name": "jsonFilePath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Google Temporary Storage Path",
+          "name": "tempBucketPath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Fully-qualified Input Table ID",
+          "name": "inputTableId"
+        },
+        {
+          "widget-type": "textbox",
```

Comment:
I don't think this should be a textbox. Check the widget for Database plugin, I believe it has an Import Query property as well